From: Mattia Rizzolo <mattia@mapreri.org>
Date: Tue, 8 Jul 2014 14:14:42 +0200
Subject: Fix_FTBFS_due_to_tests

Fix FTBFS in the test suite, to let the tests run.

Last-Update: 2014-07-02

Signed-off-by: Matteo F. Vescovi <mfvescovi@gmail.com>
---
 src/cxxtests.cpp | 124 +++++++++++++++++++++++++++----------------------------
 1 file changed, 62 insertions(+), 62 deletions(-)

diff --git a/src/cxxtests.cpp b/src/cxxtests.cpp
index 7324466..76ff6a7 100644
--- a/src/cxxtests.cpp
+++ b/src/cxxtests.cpp
@@ -23,12 +23,12 @@ int main( int argc, char *argv[] )
 #include <cxxtest/TestRunner.h>
 #include <cxxtest/RealDescriptions.h>
 
-#include "../../src/attributes-test.h"
+#include "attributes-test.h"
 
 static AttributesTest *suite_AttributesTest = 0;
 
 static CxxTest::List Tests_AttributesTest = { 0, 0 };
-CxxTest::DynamicSuiteDescription<AttributesTest> suiteDescription_AttributesTest( "../../src/attributes-test.h", 13, "AttributesTest", Tests_AttributesTest, suite_AttributesTest, 24, 25 );
+CxxTest::DynamicSuiteDescription<AttributesTest> suiteDescription_AttributesTest( "attributes-test.h", 13, "AttributesTest", Tests_AttributesTest, suite_AttributesTest, 24, 25 );
 
 static class TestDescription_AttributesTest_testAttributes : public CxxTest::RealTestDescription {
 public:
@@ -36,12 +36,12 @@ public:
  void runTest() { if ( suite_AttributesTest ) suite_AttributesTest->testAttributes(); }
 } testDescription_AttributesTest_testAttributes;
 
-#include "../../src/color-profile-test.h"
+#include "color-profile-test.h"
 
 static ColorProfileTest *suite_ColorProfileTest = 0;
 
 static CxxTest::List Tests_ColorProfileTest = { 0, 0 };
-CxxTest::DynamicSuiteDescription<ColorProfileTest> suiteDescription_ColorProfileTest( "../../src/color-profile-test.h", 13, "ColorProfileTest", Tests_ColorProfileTest, suite_ColorProfileTest, 45, 51 );
+CxxTest::DynamicSuiteDescription<ColorProfileTest> suiteDescription_ColorProfileTest( "color-profile-test.h", 13, "ColorProfileTest", Tests_ColorProfileTest, suite_ColorProfileTest, 45, 51 );
 
 static class TestDescription_ColorProfileTest_testSetRenderingIntent : public CxxTest::RealTestDescription {
 public:
@@ -61,12 +61,12 @@ public:
  void runTest() { if ( suite_ColorProfileTest ) suite_ColorProfileTest->testSetName(); }
 } testDescription_ColorProfileTest_testSetName;
 
-#include "../../src/dir-util-test.h"
+#include "dir-util-test.h"
 
 static DirUtilTest suite_DirUtilTest;
 
 static CxxTest::List Tests_DirUtilTest = { 0, 0 };
-CxxTest::StaticSuiteDescription suiteDescription_DirUtilTest( "../../src/dir-util-test.h", 9, "DirUtilTest", suite_DirUtilTest, Tests_DirUtilTest );
+CxxTest::StaticSuiteDescription suiteDescription_DirUtilTest( "dir-util-test.h", 9, "DirUtilTest", suite_DirUtilTest, Tests_DirUtilTest );
 
 static class TestDescription_DirUtilTest_testBase : public CxxTest::RealTestDescription {
 public:
@@ -74,12 +74,12 @@ public:
  void runTest() { suite_DirUtilTest.testBase(); }
 } testDescription_DirUtilTest_testBase;
 
-#include "../../src/extract-uri-test.h"
+#include "extract-uri-test.h"
 
 static ExtractURITest suite_ExtractURITest;
 
 static CxxTest::List Tests_ExtractURITest = { 0, 0 };
-CxxTest::StaticSuiteDescription suiteDescription_ExtractURITest( "../../src/extract-uri-test.h", 9, "ExtractURITest", suite_ExtractURITest, Tests_ExtractURITest );
+CxxTest::StaticSuiteDescription suiteDescription_ExtractURITest( "extract-uri-test.h", 9, "ExtractURITest", suite_ExtractURITest, Tests_ExtractURITest );
 
 static class TestDescription_ExtractURITest_testBase : public CxxTest::RealTestDescription {
 public:
@@ -99,12 +99,12 @@ public:
  void runTest() { suite_ExtractURITest.testQuoted(); }
 } testDescription_ExtractURITest_testQuoted;
 
-#include "../../src/marker-test.h"
+#include "marker-test.h"
 
 static MarkerTest suite_MarkerTest;
 
 static CxxTest::List Tests_MarkerTest = { 0, 0 };
-CxxTest::StaticSuiteDescription suiteDescription_MarkerTest( "../../src/marker-test.h", 14, "MarkerTest", suite_MarkerTest, Tests_MarkerTest );
+CxxTest::StaticSuiteDescription suiteDescription_MarkerTest( "marker-test.h", 14, "MarkerTest", suite_MarkerTest, Tests_MarkerTest );
 
 static class TestDescription_MarkerTest_testMarkerLoc : public CxxTest::RealTestDescription {
 public:
@@ -112,12 +112,12 @@ public:
  void runTest() { suite_MarkerTest.testMarkerLoc(); }
 } testDescription_MarkerTest_testMarkerLoc;
 
-#include "../../src/mod360-test.h"
+#include "mod360-test.h"
 
 static Mod360Test suite_Mod360Test;
 
 static CxxTest::List Tests_Mod360Test = { 0, 0 };
-CxxTest::StaticSuiteDescription suiteDescription_Mod360Test( "../../src/mod360-test.h", 12, "Mod360Test", suite_Mod360Test, Tests_Mod360Test );
+CxxTest::StaticSuiteDescription suiteDescription_Mod360Test( "mod360-test.h", 12, "Mod360Test", suite_Mod360Test, Tests_Mod360Test );
 
 static class TestDescription_Mod360Test_testMod360 : public CxxTest::RealTestDescription {
 public:
@@ -125,12 +125,12 @@ public:
  void runTest() { suite_Mod360Test.testMod360(); }
 } testDescription_Mod360Test_testMod360;
 
-#include "../../src/round-test.h"
+#include "round-test.h"
 
 static RoundTest *suite_RoundTest = 0;
 
 static CxxTest::List Tests_RoundTest = { 0, 0 };
-CxxTest::DynamicSuiteDescription<RoundTest> suiteDescription_RoundTest( "../../src/round-test.h", 10, "RoundTest", Tests_RoundTest, suite_RoundTest, 50, 51 );
+CxxTest::DynamicSuiteDescription<RoundTest> suiteDescription_RoundTest( "round-test.h", 10, "RoundTest", Tests_RoundTest, suite_RoundTest, 50, 51 );
 
 static class TestDescription_RoundTest_testNonNegRound : public CxxTest::RealTestDescription {
 public:
@@ -144,12 +144,12 @@ public:
  void runTest() { if ( suite_RoundTest ) suite_RoundTest->testNonPosRoung(); }
 } testDescription_RoundTest_testNonPosRoung;
 
-#include "../../src/preferences-test.h"
+#include "preferences-test.h"
 
 static PreferencesTest suite_PreferencesTest;
 
 static CxxTest::List Tests_PreferencesTest = { 0, 0 };
-CxxTest::StaticSuiteDescription suiteDescription_PreferencesTest( "../../src/preferences-test.h", 29, "PreferencesTest", suite_PreferencesTest, Tests_PreferencesTest );
+CxxTest::StaticSuiteDescription suiteDescription_PreferencesTest( "preferences-test.h", 29, "PreferencesTest", suite_PreferencesTest, Tests_PreferencesTest );
 
 static class TestDescription_PreferencesTest_testStartingState : public CxxTest::RealTestDescription {
 public:
@@ -193,12 +193,12 @@ public:
  void runTest() { suite_PreferencesTest.testPreferencesEntryMethods(); }
 } testDescription_PreferencesTest_testPreferencesEntryMethods;
 
-#include "../../src/sp-style-elem-test.h"
+#include "sp-style-elem-test.h"
 
 static SPStyleElemTest *suite_SPStyleElemTest = 0;
 
 static CxxTest::List Tests_SPStyleElemTest = { 0, 0 };
-CxxTest::DynamicSuiteDescription<SPStyleElemTest> suiteDescription_SPStyleElemTest( "../../src/sp-style-elem-test.h", 12, "SPStyleElemTest", Tests_SPStyleElemTest, suite_SPStyleElemTest, 43, 48 );
+CxxTest::DynamicSuiteDescription<SPStyleElemTest> suiteDescription_SPStyleElemTest( "sp-style-elem-test.h", 12, "SPStyleElemTest", Tests_SPStyleElemTest, suite_SPStyleElemTest, 43, 48 );
 
 static class TestDescription_SPStyleElemTest_testSetType : public CxxTest::RealTestDescription {
 public:
@@ -224,12 +224,12 @@ public:
  void runTest() { if ( suite_SPStyleElemTest ) suite_SPStyleElemTest->testReadContent(); }
 } testDescription_SPStyleElemTest_testReadContent;
 
-#include "../../src/style-test.h"
+#include "style-test.h"
 
 static StyleTest *suite_StyleTest = 0;
 
 static CxxTest::List Tests_StyleTest = { 0, 0 };
-CxxTest::DynamicSuiteDescription<StyleTest> suiteDescription_StyleTest( "../../src/style-test.h", 11, "StyleTest", Tests_StyleTest, suite_StyleTest, 37, 43 );
+CxxTest::DynamicSuiteDescription<StyleTest> suiteDescription_StyleTest( "style-test.h", 11, "StyleTest", Tests_StyleTest, suite_StyleTest, 37, 43 );
 
 static class TestDescription_StyleTest_testOne : public CxxTest::RealTestDescription {
 public:
@@ -237,12 +237,12 @@ public:
  void runTest() { if ( suite_StyleTest ) suite_StyleTest->testOne(); }
 } testDescription_StyleTest_testOne;
 
-#include "../../src/verbs-test.h"
+#include "verbs-test.h"
 
 static VerbsTest suite_VerbsTest;
 
 static CxxTest::List Tests_VerbsTest = { 0, 0 };
-CxxTest::StaticSuiteDescription suiteDescription_VerbsTest( "../../src/verbs-test.h", 7, "VerbsTest", suite_VerbsTest, Tests_VerbsTest );
+CxxTest::StaticSuiteDescription suiteDescription_VerbsTest( "verbs-test.h", 7, "VerbsTest", suite_VerbsTest, Tests_VerbsTest );
 
 static class TestDescription_VerbsTest_testEnumLength : public CxxTest::RealTestDescription {
 public:
@@ -262,12 +262,12 @@ public:
  void runTest() { suite_VerbsTest.testFetch(); }
 } testDescription_VerbsTest_testFetch;
 
-#include "../../src/display/curve-test.h"
+#include "display/curve-test.h"
 
 static CurveTest *suite_CurveTest = 0;
 
 static CxxTest::List Tests_CurveTest = { 0, 0 };
-CxxTest::DynamicSuiteDescription<CurveTest> suiteDescription_CurveTest( "../../src/display/curve-test.h", 8, "CurveTest", Tests_CurveTest, suite_CurveTest, 35, 36 );
+CxxTest::DynamicSuiteDescription<CurveTest> suiteDescription_CurveTest( "display/curve-test.h", 8, "CurveTest", Tests_CurveTest, suite_CurveTest, 35, 36 );
 
 static class TestDescription_CurveTest_testGetSegmentCount : public CxxTest::RealTestDescription {
 public:
@@ -329,12 +329,12 @@ public:
  void runTest() { if ( suite_CurveTest ) suite_CurveTest->testPenultimatePoint(); }
 } testDescription_CurveTest_testPenultimatePoint;
 
-#include "../../src/helper/units-test.h"
+#include "helper/units-test.h"
 
 static UnitsTest *suite_UnitsTest = 0;
 
 static CxxTest::List Tests_UnitsTest = { 0, 0 };
-CxxTest::DynamicSuiteDescription<UnitsTest> suiteDescription_UnitsTest( "../../src/helper/units-test.h", 7, "UnitsTest", Tests_UnitsTest, suite_UnitsTest, 17, 18 );
+CxxTest::DynamicSuiteDescription<UnitsTest> suiteDescription_UnitsTest( "helper/units-test.h", 7, "UnitsTest", Tests_UnitsTest, suite_UnitsTest, 17, 18 );
 
 static class TestDescription_UnitsTest_testConversions : public CxxTest::RealTestDescription {
 public:
@@ -348,12 +348,12 @@ public:
  void runTest() { if ( suite_UnitsTest ) suite_UnitsTest->testUnitTable(); }
 } testDescription_UnitsTest_testUnitTable;
 
-#include "../../src/libnr/in-svg-plane-test.h"
+#include "libnr/in-svg-plane-test.h"
 
 static InSvgPlaneTest *suite_InSvgPlaneTest = 0;
 
 static CxxTest::List Tests_InSvgPlaneTest = { 0, 0 };
-CxxTest::DynamicSuiteDescription<InSvgPlaneTest> suiteDescription_InSvgPlaneTest( "../../src/libnr/in-svg-plane-test.h", 9, "InSvgPlaneTest", Tests_InSvgPlaneTest, suite_InSvgPlaneTest, 31, 32 );
+CxxTest::DynamicSuiteDescription<InSvgPlaneTest> suiteDescription_InSvgPlaneTest( "libnr/in-svg-plane-test.h", 9, "InSvgPlaneTest", Tests_InSvgPlaneTest, suite_InSvgPlaneTest, 31, 32 );
 
 static class TestDescription_InSvgPlaneTest_testInSvgPlane : public CxxTest::RealTestDescription {
 public:
@@ -361,12 +361,12 @@ public:
  void runTest() { if ( suite_InSvgPlaneTest ) suite_InSvgPlaneTest->testInSvgPlane(); }
 } testDescription_InSvgPlaneTest_testInSvgPlane;
 
-#include "../../src/libnr/nr-compose-test.h"
+#include "libnr/nr-compose-test.h"
 
 static NrComposeTest *suite_NrComposeTest = 0;
 
 static CxxTest::List Tests_NrComposeTest = { 0, 0 };
-CxxTest::DynamicSuiteDescription<NrComposeTest> suiteDescription_NrComposeTest( "../../src/libnr/nr-compose-test.h", 35, "NrComposeTest", Tests_NrComposeTest, suite_NrComposeTest, 110, 111 );
+CxxTest::DynamicSuiteDescription<NrComposeTest> suiteDescription_NrComposeTest( "libnr/nr-compose-test.h", 35, "NrComposeTest", Tests_NrComposeTest, suite_NrComposeTest, 110, 111 );
 
 static class TestDescription_NrComposeTest_testnr_R8G8B8A8_N_EMPTY_R8G8B8A8_N : public CxxTest::RealTestDescription {
 public:
@@ -518,12 +518,12 @@ public:
  void runTest() { if ( suite_NrComposeTest ) suite_NrComposeTest->testnr_R8G8B8_R8G8B8_R8G8B8A8_P_A8(); }
 } testDescription_NrComposeTest_testnr_R8G8B8_R8G8B8_R8G8B8A8_P_A8;
 
-#include "../../src/libnr/nr-matrix-test.h"
+#include "libnr/nr-matrix-test.h"
 
 static NrMatrixTest *suite_NrMatrixTest = 0;
 
 static CxxTest::List Tests_NrMatrixTest = { 0, 0 };
-CxxTest::DynamicSuiteDescription<NrMatrixTest> suiteDescription_NrMatrixTest( "../../src/libnr/nr-matrix-test.h", 23, "NrMatrixTest", Tests_NrMatrixTest, suite_NrMatrixTest, 44, 45 );
+CxxTest::DynamicSuiteDescription<NrMatrixTest> suiteDescription_NrMatrixTest( "libnr/nr-matrix-test.h", 23, "NrMatrixTest", Tests_NrMatrixTest, suite_NrMatrixTest, 44, 45 );
 
 static class TestDescription_NrMatrixTest_testCtorsAssignmentOp : public CxxTest::RealTestDescription {
 public:
@@ -585,12 +585,12 @@ public:
  void runTest() { if ( suite_NrMatrixTest ) suite_NrMatrixTest->testMatrixStarScale(); }
 } testDescription_NrMatrixTest_testMatrixStarScale;
 
-#include "../../src/libnr/nr-point-fns-test.h"
+#include "libnr/nr-point-fns-test.h"
 
 static NrPointFnsTest *suite_NrPointFnsTest = 0;
 
 static CxxTest::List Tests_NrPointFnsTest = { 0, 0 };
-CxxTest::DynamicSuiteDescription<NrPointFnsTest> suiteDescription_NrPointFnsTest( "../../src/libnr/nr-point-fns-test.h", 12, "NrPointFnsTest", Tests_NrPointFnsTest, suite_NrPointFnsTest, 37, 38 );
+CxxTest::DynamicSuiteDescription<NrPointFnsTest> suiteDescription_NrPointFnsTest( "libnr/nr-point-fns-test.h", 12, "NrPointFnsTest", Tests_NrPointFnsTest, suite_NrPointFnsTest, 37, 38 );
 
 static class TestDescription_NrPointFnsTest_testL1 : public CxxTest::RealTestDescription {
 public:
@@ -634,12 +634,12 @@ public:
  void runTest() { if ( suite_NrPointFnsTest ) suite_NrPointFnsTest->testIsUnitVector(); }
 } testDescription_NrPointFnsTest_testIsUnitVector;
 
-#include "../../src/libnr/nr-rotate-test.h"
+#include "libnr/nr-rotate-test.h"
 
 static NrRotateTest *suite_NrRotateTest = 0;
 
 static CxxTest::List Tests_NrRotateTest = { 0, 0 };
-CxxTest::DynamicSuiteDescription<NrRotateTest> suiteDescription_NrRotateTest( "../../src/libnr/nr-rotate-test.h", 13, "NrRotateTest", Tests_NrRotateTest, suite_NrRotateTest, 29, 30 );
+CxxTest::DynamicSuiteDescription<NrRotateTest> suiteDescription_NrRotateTest( "libnr/nr-rotate-test.h", 13, "NrRotateTest", Tests_NrRotateTest, suite_NrRotateTest, 29, 30 );
 
 static class TestDescription_NrRotateTest_testCtorsCompares : public CxxTest::RealTestDescription {
 public:
@@ -677,12 +677,12 @@ public:
  void runTest() { if ( suite_NrRotateTest ) suite_NrRotateTest->testOpDivRotateRotate(); }
 } testDescription_NrRotateTest_testOpDivRotateRotate;
 
-#include "../../src/libnr/nr-rotate-fns-test.h"
+#include "libnr/nr-rotate-fns-test.h"
 
 static NrRotateFnsTest *suite_NrRotateFnsTest = 0;
 
 static CxxTest::List Tests_NrRotateFnsTest = { 0, 0 };
-CxxTest::DynamicSuiteDescription<NrRotateFnsTest> suiteDescription_NrRotateFnsTest( "../../src/libnr/nr-rotate-fns-test.h", 8, "NrRotateFnsTest", Tests_NrRotateFnsTest, suite_NrRotateFnsTest, 19, 20 );
+CxxTest::DynamicSuiteDescription<NrRotateFnsTest> suiteDescription_NrRotateFnsTest( "libnr/nr-rotate-fns-test.h", 8, "NrRotateFnsTest", Tests_NrRotateFnsTest, suite_NrRotateFnsTest, 19, 20 );
 
 static class TestDescription_NrRotateFnsTest_testRotateDegrees : public CxxTest::RealTestDescription {
 public:
@@ -690,12 +690,12 @@ public:
  void runTest() { if ( suite_NrRotateFnsTest ) suite_NrRotateFnsTest->testRotateDegrees(); }
 } testDescription_NrRotateFnsTest_testRotateDegrees;
 
-#include "../../src/libnr/nr-scale-test.h"
+#include "libnr/nr-scale-test.h"
 
 static NrScaleTest *suite_NrScaleTest = 0;
 
 static CxxTest::List Tests_NrScaleTest = { 0, 0 };
-CxxTest::DynamicSuiteDescription<NrScaleTest> suiteDescription_NrScaleTest( "../../src/libnr/nr-scale-test.h", 6, "NrScaleTest", Tests_NrScaleTest, suite_NrScaleTest, 20, 21 );
+CxxTest::DynamicSuiteDescription<NrScaleTest> suiteDescription_NrScaleTest( "libnr/nr-scale-test.h", 6, "NrScaleTest", Tests_NrScaleTest, suite_NrScaleTest, 20, 21 );
 
 static class TestDescription_NrScaleTest_testXY_CtorArrayOperator : public CxxTest::RealTestDescription {
 public:
@@ -739,12 +739,12 @@ public:
  void runTest() { if ( suite_NrScaleTest ) suite_NrScaleTest->testOpDivScaleScale(); }
 } testDescription_NrScaleTest_testOpDivScaleScale;
 
-#include "../../src/libnr/nr-translate-test.h"
+#include "libnr/nr-translate-test.h"
 
 static NrTranslateTest *suite_NrTranslateTest = 0;
 
 static CxxTest::List Tests_NrTranslateTest = { 0, 0 };
-CxxTest::DynamicSuiteDescription<NrTranslateTest> suiteDescription_NrTranslateTest( "../../src/libnr/nr-translate-test.h", 11, "NrTranslateTest", Tests_NrTranslateTest, suite_NrTranslateTest, 28, 29 );
+CxxTest::DynamicSuiteDescription<NrTranslateTest> suiteDescription_NrTranslateTest( "libnr/nr-translate-test.h", 11, "NrTranslateTest", Tests_NrTranslateTest, suite_NrTranslateTest, 28, 29 );
 
 static class TestDescription_NrTranslateTest_testCtorsArrayOperator : public CxxTest::RealTestDescription {
 public:
@@ -776,12 +776,12 @@ public:
  void runTest() { if ( suite_NrTranslateTest ) suite_NrTranslateTest->testIdentity(); }
 } testDescription_NrTranslateTest_testIdentity;
 
-#include "../../src/libnr/nr-types-test.h"
+#include "libnr/nr-types-test.h"
 
 static NrTypesTest *suite_NrTypesTest = 0;
 
 static CxxTest::List Tests_NrTypesTest = { 0, 0 };
-CxxTest::DynamicSuiteDescription<NrTypesTest> suiteDescription_NrTypesTest( "../../src/libnr/nr-types-test.h", 8, "NrTypesTest", Tests_NrTypesTest, suite_NrTypesTest, 23, 24 );
+CxxTest::DynamicSuiteDescription<NrTypesTest> suiteDescription_NrTypesTest( "libnr/nr-types-test.h", 8, "NrTypesTest", Tests_NrTypesTest, suite_NrTypesTest, 23, 24 );
 
 static class TestDescription_NrTypesTest_testXYValues : public CxxTest::RealTestDescription {
 public:
@@ -843,12 +843,12 @@ public:
  void runTest() { if ( suite_NrTypesTest ) suite_NrTypesTest->testNormalize(); }
 } testDescription_NrTypesTest_testNormalize;
 
-#include "../../src/svg/css-ostringstream-test.h"
+#include "svg/css-ostringstream-test.h"
 
 static CSSOStringStreamTest *suite_CSSOStringStreamTest = 0;
 
 static CxxTest::List Tests_CSSOStringStreamTest = { 0, 0 };
-CxxTest::DynamicSuiteDescription<CSSOStringStreamTest> suiteDescription_CSSOStringStreamTest( "../../src/svg/css-ostringstream-test.h", 20, "CSSOStringStreamTest", Tests_CSSOStringStreamTest, suite_CSSOStringStreamTest, 26, 27 );
+CxxTest::DynamicSuiteDescription<CSSOStringStreamTest> suiteDescription_CSSOStringStreamTest( "svg/css-ostringstream-test.h", 20, "CSSOStringStreamTest", Tests_CSSOStringStreamTest, suite_CSSOStringStreamTest, 26, 27 );
 
 static class TestDescription_CSSOStringStreamTest_testFloats : public CxxTest::RealTestDescription {
 public:
@@ -868,12 +868,12 @@ public:
  void runTest() { if ( suite_CSSOStringStreamTest ) suite_CSSOStringStreamTest->testConcat(); }
 } testDescription_CSSOStringStreamTest_testConcat;
 
-#include "../../src/svg/stringstream-test.h"
+#include "svg/stringstream-test.h"
 
 static StringStreamTest *suite_StringStreamTest = 0;
 
 static CxxTest::List Tests_StringStreamTest = { 0, 0 };
-CxxTest::DynamicSuiteDescription<StringStreamTest> suiteDescription_StringStreamTest( "../../src/svg/stringstream-test.h", 21, "StringStreamTest", Tests_StringStreamTest, suite_StringStreamTest, 27, 28 );
+CxxTest::DynamicSuiteDescription<StringStreamTest> suiteDescription_StringStreamTest( "svg/stringstream-test.h", 21, "StringStreamTest", Tests_StringStreamTest, suite_StringStreamTest, 27, 28 );
 
 static class TestDescription_StringStreamTest_testFloats : public CxxTest::RealTestDescription {
 public:
@@ -893,12 +893,12 @@ public:
  void runTest() { if ( suite_StringStreamTest ) suite_StringStreamTest->testConcat(); }
 } testDescription_StringStreamTest_testConcat;
 
-#include "../../src/svg/svg-affine-test.h"
+#include "svg/svg-affine-test.h"
 
 static SvgAffineTest *suite_SvgAffineTest = 0;
 
 static CxxTest::List Tests_SvgAffineTest = { 0, 0 };
-CxxTest::DynamicSuiteDescription<SvgAffineTest> suiteDescription_SvgAffineTest( "../../src/svg/svg-affine-test.h", 12, "SvgAffineTest", Tests_SvgAffineTest, suite_SvgAffineTest, 46, 47 );
+CxxTest::DynamicSuiteDescription<SvgAffineTest> suiteDescription_SvgAffineTest( "svg/svg-affine-test.h", 12, "SvgAffineTest", Tests_SvgAffineTest, suite_SvgAffineTest, 46, 47 );
 
 static class TestDescription_SvgAffineTest_testReadIdentity : public CxxTest::RealTestDescription {
 public:
@@ -984,12 +984,12 @@ public:
  void runTest() { if ( suite_SvgAffineTest ) suite_SvgAffineTest->testReadFailures(); }
 } testDescription_SvgAffineTest_testReadFailures;
 
-#include "../../src/svg/svg-color-test.h"
+#include "svg/svg-color-test.h"
 
 static SVGColorTest *suite_SVGColorTest = 0;
 
 static CxxTest::List Tests_SVGColorTest = { 0, 0 };
-CxxTest::DynamicSuiteDescription<SVGColorTest> suiteDescription_SVGColorTest( "../../src/svg/svg-color-test.h", 9, "SVGColorTest", Tests_SVGColorTest, suite_SVGColorTest, 35, 36 );
+CxxTest::DynamicSuiteDescription<SVGColorTest> suiteDescription_SVGColorTest( "svg/svg-color-test.h", 9, "SVGColorTest", Tests_SVGColorTest, suite_SVGColorTest, 35, 36 );
 
 static class TestDescription_SVGColorTest_testWrite : public CxxTest::RealTestDescription {
 public:
@@ -1009,12 +1009,12 @@ public:
  void runTest() { if ( suite_SVGColorTest ) suite_SVGColorTest->testIccColor(); }
 } testDescription_SVGColorTest_testIccColor;
 
-#include "../../src/svg/svg-length-test.h"
+#include "svg/svg-length-test.h"
 
 static SvgLengthTest *suite_SvgLengthTest = 0;
 
 static CxxTest::List Tests_SvgLengthTest = { 0, 0 };
-CxxTest::DynamicSuiteDescription<SvgLengthTest> suiteDescription_SvgLengthTest( "../../src/svg/svg-length-test.h", 10, "SvgLengthTest", Tests_SvgLengthTest, suite_SvgLengthTest, 29, 30 );
+CxxTest::DynamicSuiteDescription<SvgLengthTest> suiteDescription_SvgLengthTest( "svg/svg-length-test.h", 10, "SvgLengthTest", Tests_SvgLengthTest, suite_SvgLengthTest, 29, 30 );
 
 static class TestDescription_SvgLengthTest_testRead : public CxxTest::RealTestDescription {
 public:
@@ -1058,12 +1058,12 @@ public:
  void runTest() { if ( suite_SvgLengthTest ) suite_SvgLengthTest->testPlaces(); }
 } testDescription_SvgLengthTest_testPlaces;
 
-#include "../../src/svg/svg-path-geom-test.h"
+#include "svg/svg-path-geom-test.h"
 
 static SvgPathGeomTest *suite_SvgPathGeomTest = 0;
 
 static CxxTest::List Tests_SvgPathGeomTest = { 0, 0 };
-CxxTest::DynamicSuiteDescription<SvgPathGeomTest> suiteDescription_SvgPathGeomTest( "../../src/svg/svg-path-geom-test.h", 13, "SvgPathGeomTest", Tests_SvgPathGeomTest, suite_SvgPathGeomTest, 69, 70 );
+CxxTest::DynamicSuiteDescription<SvgPathGeomTest> suiteDescription_SvgPathGeomTest( "svg/svg-path-geom-test.h", 13, "SvgPathGeomTest", Tests_SvgPathGeomTest, suite_SvgPathGeomTest, 69, 70 );
 
 static class TestDescription_SvgPathGeomTest_testReadRectanglesAbsoluteClosed : public CxxTest::RealTestDescription {
 public:
@@ -1179,12 +1179,12 @@ public:
  void runTest() { if ( suite_SvgPathGeomTest ) suite_SvgPathGeomTest->testMinexpPrecision(); }
 } testDescription_SvgPathGeomTest_testMinexpPrecision;
 
-#include "../../src/xml/repr-action-test.h"
+#include "xml/repr-action-test.h"
 
 static XmlReprActionTest *suite_XmlReprActionTest = 0;
 
 static CxxTest::List Tests_XmlReprActionTest = { 0, 0 };
-CxxTest::DynamicSuiteDescription<XmlReprActionTest> suiteDescription_XmlReprActionTest( "../../src/xml/repr-action-test.h", 11, "XmlReprActionTest", Tests_XmlReprActionTest, suite_XmlReprActionTest, 33, 34 );
+CxxTest::DynamicSuiteDescription<XmlReprActionTest> suiteDescription_XmlReprActionTest( "xml/repr-action-test.h", 11, "XmlReprActionTest", Tests_XmlReprActionTest, suite_XmlReprActionTest, 33, 34 );
 
 static class TestDescription_XmlReprActionTest_testRollbackOfNodeAddition : public CxxTest::RealTestDescription {
 public:
@@ -1204,12 +1204,12 @@ public:
  void runTest() { if ( suite_XmlReprActionTest ) suite_XmlReprActionTest->testRollbackOfNodeReordering(); }
 } testDescription_XmlReprActionTest_testRollbackOfNodeReordering;
 
-#include "../../src/xml/quote-test.h"
+#include "xml/quote-test.h"
 
 static XmlQuoteTest *suite_XmlQuoteTest = 0;
 
 static CxxTest::List Tests_XmlQuoteTest = { 0, 0 };
-CxxTest::DynamicSuiteDescription<XmlQuoteTest> suiteDescription_XmlQuoteTest( "../../src/xml/quote-test.h", 15, "XmlQuoteTest", Tests_XmlQuoteTest, suite_XmlQuoteTest, 26, 27 );
+CxxTest::DynamicSuiteDescription<XmlQuoteTest> suiteDescription_XmlQuoteTest( "xml/quote-test.h", 15, "XmlQuoteTest", Tests_XmlQuoteTest, suite_XmlQuoteTest, 26, 27 );
 
 static class TestDescription_XmlQuoteTest_testXmlQuotedStrlen : public CxxTest::RealTestDescription {
 public:
@@ -1223,12 +1223,12 @@ public:
  void runTest() { if ( suite_XmlQuoteTest ) suite_XmlQuoteTest->testXmlQuoteStrdup(); }
 } testDescription_XmlQuoteTest_testXmlQuoteStrdup;
 
-#include "../../src/util/list-container-test.h"
+#include "util/list-container-test.h"
 
 static ListContainerTest *suite_ListContainerTest = 0;
 
 static CxxTest::List Tests_ListContainerTest = { 0, 0 };
-CxxTest::DynamicSuiteDescription<ListContainerTest> suiteDescription_ListContainerTest( "../../src/util/list-container-test.h", 28, "ListContainerTest", Tests_ListContainerTest, suite_ListContainerTest, 38, 39 );
+CxxTest::DynamicSuiteDescription<ListContainerTest> suiteDescription_ListContainerTest( "util/list-container-test.h", 28, "ListContainerTest", Tests_ListContainerTest, suite_ListContainerTest, 38, 39 );
 
 static class TestDescription_ListContainerTest_testRangeConstructor : public CxxTest::RealTestDescription {
 public:
